packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1208,12 +1208,25 @@ class _SelectionHandleOverlayState extends State<_SelectionHandleOverlay> with S
           alignment: Alignment.topLeft,
           width: interactiveRect.width,
           height: interactiveRect.height,
-          child: GestureDetector(
+          child: RawGestureDetector(
             behavior: HitTestBehavior.translucent,
-            dragStartBehavior: widget.dragStartBehavior,
-            onPanStart: widget.onSelectionHandleDragStart,
-            onPanUpdate: widget.onSelectionHandleDragUpdate,
-            onPanEnd: widget.onSelectionHandleDragEnd,
+            gestures: <Type, GestureRecognizerFactory>{
+              PanGestureRecognizer: GestureRecognizerFactoryWithHandlers<PanGestureRecognizer>(
+                () => PanGestureRecognizer(
+                  debugOwner: this,
+                  // Selection handles can only be dragged by touch. Mouse
+                  // events on the handle will select the text behind.
+                  supportedDevices: <PointerDeviceKind>{ PointerDeviceKind.touch },
```

Comment:
what about stylus?

and there is the wild card, PointerDeviceKind.unknown. as far as I know, adb comand and a11y command may use  `unknown`. I think a11y is probably fine to be excluded. adb commands are usually used during integration test, which may expect to drag the selection handle. I am not sure whether that would be a valid use case though

Reply:
That makes sense, let me try adding `stylus` and `unknown` and make sure the tests still pass here.

Reply:
I added them plus separate tests for touch, stylus, and unknown (and the existing mouse test).  Ready for re-review.